test/rest-api/response/body/body-stream.test.ts

```diff
@@ -0,0 +1,17 @@
+import * as path from 'path'
+import { pageWith } from 'page-with'
+
+test('responds with a ReadableStream response body', async () => {
+  const { request } = await pageWith({
+    example: path.resolve(__dirname, 'body-stream.mocks.ts'),
+  })
+
+  const res = await request('/sse')
+  const status = res.status()
+  const headers = await res.allHeaders()
+  const text = await res.text()
```

Comment:
We need to ensure that `res.text()` actually verifies that you can read that response stream once more on the usage surface (assert that we're correctly piping the original response stream into `PassThrough` in the worker).